src/apps/api/views/competitions.py

```diff
@@ -337,6 +337,30 @@ def create_dump(self, request, pk=None):
         serializer = CompetitionCreationTaskStatusSerializer({"status": "Success. Competition dump is being created."})
         return Response(serializer.data, status=201)
 
+    def run_new_task_submissions(self, phase, tasks):
+        tasks_id = set([task.id for task in tasks])
+        submissions = phase.submissions.prefetch_related("children").all()
+        parents = lambda s: s.has_children
+
+        for submission in filter(parents, submissions):
+            child_tasks = set([sub.task for sub in submission.children.all()])
+            missing_tasks = tasks_id - set([task.id for task in child_tasks])
+            missing_task_fitler = lambda t: t.id in missing_tasks
+            for task in filter(missing_task_fitler, tasks):
+                print(f'\nRunning Task({task}) for {submission}\n')
+                sub = Submission(
+                    owner=submission.owner,
+                    phase=submission.phase,
+                    parent=submission,
+                    task=task,
+                    appear_on_leaderboards=submission.appear_on_leaderboards,
```

Comment:
This might cause issues with tasks that are not on the leaderboard. Might be better to get the leaderboard from a child submission?